Rakefile

```diff
@@ -53,7 +53,8 @@ def rake(*arguments)
 end
 
 task :test do
-  ruby("test/run-test.rb")
+  args_string = ENV["TESTOPTS"]
+  ruby("test/run-test.rb #{args_string}")
```

Comment:
Can we do this in `test/run-test.rb` instead?

Reply:
If we handle `ENV["TESTOPTS"]` in `test/run-test.rb`, it's not passed as command line arguments. It's passed as an environment variable. So the command line doesn't have `--parallel`.

Reply:
Thanks for your reply. It makes sense.